### PR TITLE
PD-273 Reorganize and Update Support Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit reorganizes getting support, filing issue tickets, and other related ticket content. 

It creates a new expand section in /GettingStarted/GetSupport and moves and/or creates new articles with moved content. 

It updates the /Contributing/IssueReporting/_index.md file to introduce the IssueReporting content moved into a new IssueReporting.md article. 

It creates new snippets to create a scale debug, FileIssueSCALE.md. 

It moves the Add License and Proactive Support content into a new AddLicensProactiveSupport.md article in the /SCALETutorials/SystemSettings/General/ folder.

Merge this content in to Master.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by micjohnson777** [comment]
Typo in the PR title fixed, it is PD-273 not PD-237
I am working on more changes in the migration section under PD-237, which is going to create a more organized and comprehensive section similar to the Install and Configure sections. This PR changes content by adding a new article and snippet.

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+* The current service configuration if the bug is service-related to help developers replicate your system.
+
+* Any additional details you think can help the developer investigate.
+
+In addition to the above information, please also provide: 
+
+* A TrueNAS debug file. This is required to help developers identify and troubleshoot issues on your system. 
+
+  Include a console log if the error is a web interface issue. 
+  If unsure how to save these, see the [Adding a Debug File to a Report](#adding-a-debug-file-to-a-report) and [Web Console Log](#obtaining-web-console-logs) sections in this article.
```
Comment (clairewrtr) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:58]:
```suggestion
  If unsure how to save these, see the [Adding a Debug File to a Report](#adding-a-debug-file-to-a-jira-ticket) and [Web Console Log](#obtaining-web-console-logs) sections in this article.
```

File: static/includes/General/CreateJiraSuggestion.md.part [hunk 2]
```diff
@@ -8,6 +8,6 @@ To suggest a new feature, go to https://jira.ixsystems.com/projects/NAS/, log in
 
 ![JiraSuggestionCreate](/images/Contribute/JiraSuggestionCreate.png "Creating a new Suggestion")
 
-Briefly describe the new feature you'd like to see added in the **Summary** section.
+Briefly describe the new feature you would like to see added in the **Summary** section.
```
Comment (clairewrtr) [inline static/includes/General/CreateJiraSuggestion.md.part:11]: The "here" link in line 2 does not work.
[here](https://jira.ixsystems.com/issues/?jql=issuetype%20%3D%20Suggestion%20AND%20status%20%3D%20%22Gathering%20Interest%22)

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+![Submit Ticket](/images/Contribute/JiraCreateBottom.png "Submit Ticket")
+
+Developers review and update the ticket if/when they need additional information.
+The Jira account receives emails about the ticket status.
+Developers might request more details as they work to resolve the issue, so check the ticket periodically.
+
+### Making Feature Suggestions
+
+{{< include file="static/includes/General/CreateJiraSuggestion.md.part" markdown="true" >}}
```
Comment (clairewrtr) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:76]:
```suggestion
{{< include file="static/includes/General/CreateJiraSuggestion.md.part" markdown="true" >}}
The "here" link in this includes file does not work.
```

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
```
Comment (clairewrtr) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:77]: The "here" link in the includes file on line 76 does not work.

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+After saving the file, open the console with <i class="fa fa-bars" aria-hidden="true" title="Menu"></i> **(Menu) > More Tools > Web Developer Tools** (<kbd>Ctrl-Shift-I</kbd>) and unset *Persist Logs*.
+
+### Chrome
+
+Open the console by clicking <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> **(Options) > More Tools > Developer tools** (<kbd>Ctrl-Shift-I</kbd>).
+
+Click <i class="fa fa-cog" aria-hidden="true" title="Settings"></i> **(Preferences)** and select **Preserve log** and **Show timestamps**. Close the **Preferences** window.
+
+Leave the console open and perform the action that encounters problems. Right-click the console window. Choose ***Save as** and save the file.
```
Comment (clairewrtr) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:142]:
```suggestion
Leave the console open and perform the action that encounters problems. Right-click the console window. Choose **Save as** and save the file.
```

File: content/SCALE/GettingStarted/Configure/UIConfigurationSCALE.md [hunk 6]
```diff
@@ -45,11 +45,10 @@ To apply the license information, go to the **System Settings > General** screen
 This content applies to TrueNAS Enterprise customers only.
 
 TrueNAS SCALE Enterprise customers with Silver or Gold Coverage support contracts have the option to configure proactive support.
-{{< expand "Configuring Proactive Support" "v" >}}
-Customers with appropriate support contracts can configure **[Proactive Support]({{< relref "GetSupportSCALE.md" >}})** after they apply their system license, and after acknowledging and signing the End User License Agreement (EULA).
+
+Customers with appropriate support contracts can configure **[Proactive Support]({{< relref "AddLicenseProactiveSupport.md" >}})** after they apply their system license, and after acknowledging and signing the End User License Agreement (EULA).
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Configure/UIConfigurationSCALE.md:49]: Line 239: 
**[Updating SCALE]
Remove the double asterisk before [Updataing SCALE]

File: content/_includes/FileIssueTicketSCALE.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+* Use the **File Ticket** option on the **System Settings > Genral** screen to create a bug report in the [TrueNAS Jira Project](https://ixsystems.atlassian.net/jira/software/c/projects/NAS/issues). 
+
+The web interface provides a form to report issues without logging out of SCALE, and that prompts you to provide the information and attachments we need to assist users. 
+
+{{< hint info >}}
+New Jira tickets are not marked as **iX Private** so it is possible to search the project first to see if another user already reported the issue. 
+Each Jira ticket sends a link to a private file attachement area to safeguard user personal and private data, or if the attached files do not require privacy, attach them to the Jira ticket. All incoming tickets are triaged. If private files are attached to a new ticket, the ticket or files can be made private at that time. 
+{{< /hint >}}
+
+You must have [a Jira account](https://jira.ixsystems.com/secure/Signup!default.jspa) to create a bug ticket.
```
Comment (clairewrtr) [inline content/_includes/FileIssueTicketSCALE.md:19]: Check if this link is working.

File: content/SCALE/GettingStarted/Configure/UIConfigurationSCALE.md (hunk 6 above)
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Configure/UIConfigurationSCALE.md:49]: found an incomplete thought! good catch. fixed it in the next commit to this PR.

File: content/SCALE/SCALETutorials/SystemSettings/General/AddLicenseProactiveSupport.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+ - scalesupport
+ - scaleconfig
+---
+
+
+{{< toc >}}
+
+
+## Adding a License
+For users with a valid TrueNAS license, click **Add License**. Copy your license into the box and click **Save**.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/General/AddLicenseProactiveSupport.md:16]: fixed in the next commit to this PR.

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+* A brief, specific, description detailing the problem you encountered.
+
+* The steps to reproduce the issue. A simple list of the steps you took to see the issue is fine.
+
+* An explanation of what should have happened while taking the steps listed above.
+
+* A description of what actually happened while taking the steps listed above.
+
+* The TrueNAS software version you are using which is found on the main **Dashboard**.
```
Comment (tonyriv3) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:47]:
```suggestion
* The TrueNAS software version you are using, which is found on the main **Dashboard**.
```

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+
+### Adding a Debug File to a Jira Ticket
+Jira provides a public facing area for files that do not require privacy. 
+iXsystems also provides a secure file attachment area that only developers can access, where you can upload files with sensitive information like a system debug or screenshot of network connections.
+
+Debug files attached directly to the Jira issue ticket are visible to other users searching and reading Jira issue tickets. 
+To keep your sensitive information private, use the link in the system-generated message posted to your Jira ticket to upload files to the private attachment area. 
+Be sure to link your file uploads to your Jira ticket number before you click **Save** after uploading your file. 
+
+If you are not concerned about data privacy, you can attach the debug, core, log or other files to the Jira ticket.
```
Comment (tonyriv3) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:97]:
```suggestion
If you are not concerned about data privacy, you can attach the debug, core, log, or other files to the Jira ticket. 
```

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+
+Debug files attached directly to the Jira issue ticket are visible to other users searching and reading Jira issue tickets. 
+To keep your sensitive information private, use the link in the system-generated message posted to your Jira ticket to upload files to the private attachment area. 
+Be sure to link your file uploads to your Jira ticket number before you click **Save** after uploading your file. 
+
+If you are not concerned about data privacy, you can attach the debug, core, log or other files to the Jira ticket. 
+
+### Attaching Files to New Tickets
+
+Drag and drop public facing files into the **Attachment** box when creating a new ticket:
```
Comment (tonyriv3) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:101]:
```suggestion
Drag and drop public-facing files into the **Attachment** box when creating a new ticket:
```

File: content/Contributing/IssueReporting/JiraIssueReporting.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+
+![JiraAttachmentNew](/images/Contribute/newjiraattachments.png "NAS Project Bug Creation Form")
+
+To protect your privacy, please use the link in the system generated message to upload your debug or any screenshot with sensitive information to the private file attachment area.
```
Comment (tonyriv3) [inline content/Contributing/IssueReporting/JiraIssueReporting.md:105]:
```suggestion
To protect your privacy, please use the link in the system-generated message to upload your debug or any screenshot with sensitive information to the private file attachment area. 
```

File: content/SCALE/GettingStarted/Getting Support/FileIssueSCALE.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+After logging into Jira, select either **Bug** or **Feature** as the **Type** of ticket to create, then choose the appropriate **Category** for your request.
+
+Attach a debug file to all bug tickets. 
+Click **Attach Debug**. SCALE generates the debug file, uploads it to the private attachment area, and creates the link to the Jira ticket. Debug files give the TrueNAS team pertinent information about the system and what could be causing any issues. 
+If the debug file is too large to attach to your ticket, the following displays:
+
+![GeneralSettingsSCALEJiraTicketDebugToLarge](/images/SCALE/GeneralSettingsSCALEJiraTicketDebugToLarge.png "SCALE General Settings Debug To Large To Attach")  
+
+Provide a brief summary of the issue in **Subject**. 
+Enter much details about the issue as possible as the reason for submitting the ticket in the **Description** field.
```
Comment (tonyriv3) [inline content/SCALE/GettingStarted/Getting Support/FileIssueSCALE.md:40]:
```suggestion
Enter as many details about the issue as possible and the reason for submitting the ticket in the **Description** field. 
```

File: content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+## About TrueNAS Support
+
+{{< enterprise >}}
+Enterprise SCALE hardware customers with support contracts should contact iXsystems Support using either the **[Commercial Support](https://www.truenas.com/commercial-support/)** option on the top header of the TrueNAS Documentation Hub website, or through one of the contact options listed below.
+{{< expand "Contacting iXsystems Support" "v" >}}
+{{< include file="static/includes/General/iXsystemsSupportContact.html.part" html="true" >}}
+{{< /expand >}}
+{{< /enterprise >}}
+
+TrueNAS SCALE users are welcome to report bugs, suggest new TrueNAS features, and vote for suggested improvment in the Jira project instance.
```
Comment (tonyriv3) [inline content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md:22]:
```suggestion
TrueNAS SCALE users are welcome to report bugs, suggest new TrueNAS features, and vote for suggested improvements in the Jira project instance. 
```

File: content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+{{< expand "Contacting iXsystems Support" "v" >}}
+{{< include file="static/includes/General/iXsystemsSupportContact.html.part" html="true" >}}
+{{< /expand >}}
+{{< /enterprise >}}
+
+TrueNAS SCALE users are welcome to report bugs, suggest new TrueNAS features, and vote for suggested improvment in the Jira project instance. 
+Have questions? We recommend searching through the software documentation and community resources for answers. 
+
+Non-Enterprise SCALE customers experiencing software bugs or instability can try to find answers in the various community forums or can [file an issue ticket](https://www.truenas.com/docs/contributing/issuereporting/) through the Jira ticket reporting system for TrueNAS. The [Issue Reporting]({{< relref "/content/Contributing/IssueReporting/_index.md" >}}) article for more information what type of information to include in issue tickets in Jira, and attaching files to issue tickets.
```
Comment (tonyriv3) [inline content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md:25]:
```suggestion
Non-Enterprise SCALE customers experiencing software bugs or instability can try to find answers in the various community forums, or they can [file an issue ticket](https://www.truenas.com/docs/contributing/issuereporting/) through the Jira ticket reporting system for TrueNAS. The [Issue Reporting]({{< relref "/content/Contributing/IssueReporting/_index.md" >}}) article has more information on what type of information and attachments to include in Jira tickets.
```

File: content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+
+Support is also available through the TrueNAS community forums, blog, and Discord. 
+These options are accessible on the top header of the TrueNAS Documentation Hub website and from the links at the bottom of all articles.
+
+## Using the TrueNAS Community
+The [TrueNAS Community](https://www.truenas.com/community/) is an active online resource for asking questions, troubleshooting issues, and sharing information with other TrueNAS users.
+You must [register](https://www.truenas.com/community/register/) to post.
+We encourage new users to briefly [introduce](https://www.truenas.com/community/forums/introductions.25/) themselves and review the [forum rules](https://www.truenas.com/community/threads/forum-rules.45124/) before posting.
+
+[Community Resources](https://www.truenas.com/community/resources/) are user-contributed articles about every facet of using TrueNAS, organized into broad categories,and they incorporate a community rating system to better highlight content that the whole community has found helpful.
```
Comment (tonyriv3) [inline content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md:43]:
```suggestion
[Community Resources](https://www.truenas.com/community/resources/) are user-contributed articles about every facet of using TrueNAS organized into broad categories. They incorporate a community rating system to better highlight content that the whole community has found helpful.
```

File: content/SCALE/SCALETutorials/SystemSettings/General/AddLicenseProactiveSupport.md [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+After you finish, click **I AGREE**. The system information updates to reflect the licensing specifics for the system.
+
+![GeneralSettingsSCALESupportLicenseComplete](/images/SCALE/GeneralSettingsSCALESupportLicenseComplete.png "SCALE General Settings Support License Entry Complete")
+
+Silver and Gold level Support customers can also enable Proactive Support on their hardware to automatically notify iXsystems if an issue occurs. 
+To find more details about the different Warranty and Service Level Agreement (SLA) options available, see [iXsystems Support](https://www.ixsystems.com/support/).
+
+When the system is ready to be in production, update the status by selecting **This is a production system** and then click the **Proceed** button. This sends an email to iXsystems declaring that the system is in production. 
+
+While not required for declaring the system is in production, TrueNAS has an option to include a initial debug with the email that can assist support in the future.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/SystemSettings/General/AddLicenseProactiveSupport.md:32]:
```suggestion
While not required for declaring the system is in production, TrueNAS has the option to include an initial debug with the email that can assist support in the future.
```

File: content/_includes/CreateDebugSCALE.md [hunk 18]
```diff
@@ -0,0 +1,5 @@
+---
+---
+
+On TrueNAS SCALE systems, go to **System Settings > Advanced**, then click **Save Debug** and wait for the file to download to your local system. 
+The generating the debug file might take a few minutes to complete. After that it downloads to your system.
```
Comment (tonyriv3) [inline content/_includes/CreateDebugSCALE.md:5]:
```suggestion
On TrueNAS SCALE systems, go to **System Settings > Advanced**, then click **Save Debug** and wait for the file to download to your local system. 
Generating the debug file might take a few minutes to complete. After that, it downloads to your system.
```

File: content/_includes/FileIssueTicketSCALE.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
+---
+---
+
+TrueNAS SCALE users are encouraged to report bugs and to vote for or suggest new TrueNAS features in the project Jira instance. 
+Have questions? We recommend searching through the software documentation and community resources for answers.
```
Comment (tonyriv3) [inline content/_includes/FileIssueTicketSCALE.md:5]:
```suggestion
We encourage TrueNAS SCALE users to report bugs and to vote for or suggest new TrueNAS features in the project Jira instance. 
Have questions? We recommend searching through the software documentation and community resources for answers.
```

File: content/_includes/FileIssueTicketSCALE.md [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+If you encounter a bug or other issue while using TrueNAS SCALE, you can report issues in one of two ways:
+
+* Log into Jira and use **Create** to open a new ticket (see [Issue Reporting]({{< relref "/Content/Contributing/IssueReporting/JiraIssueReporting.md" >}}) for more information).
+* Use the **File Ticket** option on the **System Settings > Genral** screen to create a bug report in the [TrueNAS Jira Project](https://ixsystems.atlassian.net/jira/software/c/projects/NAS/issues). 
+
+The web interface provides a form to report issues without logging out of SCALE, and that prompts you to provide the information and attachments we need to assist users. 
+
+{{< hint info >}}
+New Jira tickets are not marked as **iX Private** so it is possible to search the project first to see if another user already reported the issue. 
+Each Jira ticket sends a link to a private file attachement area to safeguard user personal and private data, or if the attached files do not require privacy, attach them to the Jira ticket. All incoming tickets are triaged. If private files are attached to a new ticket, the ticket or files can be made private at that time.
```
Comment (tonyriv3) [inline content/_includes/FileIssueTicketSCALE.md:16]:
```suggestion
Each Jira ticket sends a link to a private file attachment area to safeguard user personal and private data. If the attached files do not require privacy, attach them to the Jira ticket. All incoming tickets are triaged. If private files are attached to a new ticket, the ticket or files can be made private at that time. 
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**